examples/docker_aws_lambda_example/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM fedora:latest
+
+# Install pre-reqs
+RUN dnf install wget -y
+RUN dnf install gcc -y
+
+# Install librdkafka
+RUN rpm --import https://packages.confluent.io/rpm/5.4/archive.key
+RUN dnf install librdkafka-devel -y
```

Comment:
Why is librdkafka required? It looks like the build step below uses the Go-bundled librdkafka.